src/commands/BootstrapCommand.js

```diff
@@ -49,24 +50,24 @@ export default class BootstrapCommand extends Command {
       const linkSrc = dependency.location;
       const linkDest = path.join(pkg.nodeModulesLocation, dependency.name);
 
-      this.createLinkedDependency(linkSrc, linkDest, dependency.name, done);
+      this.createLinkedDependency(linkSrc, linkDest, dependency.name, (err) => {
+        if (!err && dependency._package.bin) {
```

Comment:
i've neglected this PR in favor of PR #237, which accounts for this and also add a `.bin` getter to `Package` as you mentioned above.